### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aherbst-broad**
Update README. The bulk of the changes are me running the README through a formatter, I've called out the substantive changes in breadcrumb comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,57 +142,35 @@ Incompatible changes require incrementing the major version number. In our curre
...
+see
 [Releasing our Code](https://broadworkbench.atlassian.net/wiki/spaces/WOR/pages/2974449675/Releasing+our+Code)
 in Confluence.
 
 ## Setup
 
 ### Prerequisites:
 
-#### Postgres
-We are currently using Postgres 13.1.
```
Comment (aherbst-broad) [inline README.md:166]: I don't think running WSM with a local postgres is supported anymore (I'm unaware of anyone running in this mode). I want to centralize on a single approach for turning on the DB.

File: README.md [hunk 2]
```diff
@@ -82,20 +82,22 @@ have WRITER on the workspace, then you can write the resource.
 Private resources are available to a single member of the workspace. At the present time,
 a private resource is available only to its creator.
 
-WSM has latent support for _applications_. No applications exist at this time. The concept
-is that an application is a distinguished service account. Owners of the workspace can
-control which applications are allowed access to the workspace. If an application is given
+WSM has latent support for _applications_. [Leonardo](https://github.com/DataBiosphere/leonardo) is the only
```
Comment (aherbst-broad) [inline README.md:85]: Small change re: leo

File: README.md [hunk 3]
```diff
@@ -383,9 +374,7 @@ local WSM:
 For Broad deployment, those are:
 
 - dev: https://workspace.dsde-dev.broadinstitute.org/swagger-ui.html
-- alpha: https://workspace.dsde-alpha.broadinstitute.org/swagger-ui.html
```
Comment (aherbst-broad) [inline README.md:386]: These envs are dead.

File: README.md (hunk 1 above)
Comment (blakery) [reply README.md:166]: I don't think anyone really runs postgres outside of a docker container _anywhere_, anymore.

**Review by blakery — APPROVED** [review]

**Review by marctalbott — APPROVED** [review]

**Pull request merged**